### PR TITLE
Updates to the document

### PR DESCRIPTION
**Pull request by jonsterling**
In light of the fact that some outsiders may be looking at this document soon (due to my upcoming talk), I have made some changes.

- I have replaced the cool pseudonym with the real authors (Angiuli, Cavallo, Favonia, Sterling): we can find other ways to worship St. Facaan.

- I have made some clarificatory changes to the remark on cap/tube.

- I have improved some formatting and notation.

- I have added a few more citations.

### CODE REVIEW

File: doc/kan.tex [hunk 1]
```diff
@@ -178,8 +178,11 @@
     The strict equations lose too much information too early, such as the Kan-ness
     and universe levels. For example, it is difficult to recognize
     \[
-      \Prod{i : \DIM}{\prn{\TpSub{\TpEl{\Code{A}\prn{i}}}{\Boundary{i}}{
```
Comment (cangiuli) [inline doc/kan.tex:171]: :man_dancing:

File: doc/kan.tex [hunk 2]
```diff
@@ -463,19 +480,41 @@ \section{The universe}
...
     \item Consolidate ``cap'' and ``tube'' into one system.
-    \item Good elaboration support for cubical subtypes.
+    \item Good judgmental support for cubical subtypes.
   \end{enumerate}
-  We did both in \texttt{cooltt}.
+
+  We have done both in \texttt{cooltt}.
 \end{cool}
 
 \subsection{Rules for composite types}
```
Comment (cangiuli) [inline doc/kan.tex:520]: can we explain briefly what "cubically unstable" means?

**Comment by jonsterling** [comment]
There has been some discussion over what is meant by "over-approximate" in the remark about Cubical Agda. I wish to clarify matters.

It is not the case that this "over-approximation" is equally expressed in `primHcomp` and in `inc`. The types assigned to the operators associated with cubical subtypes are actually totally correct:

```agda
    inc : ∀ {ℓ} {A : Set ℓ} {φ} (x : A) → Sub A φ (λ _ → x)
    primSubOut : ∀ {ℓ} {A : Set ℓ} {φ : I} {u : Partial φ A} → Sub _ φ u → A
```

On the other hand, the primitive for homogeneous composition is given a type *that it does not have* (unless I am misunderstanding something): 

```agda
primHComp  : ∀ {ℓ} {A : Set ℓ} {φ : I} (u : ∀ i → Partial φ A) (a : A) → A
```

/cc @cangiuli @ecavallo

**Comment by ecavallo** [comment]
Ah, yeah, that's my mistake.

**Comment by jonsterling** [comment]
no worries! 😄

**Pull request merged**